scripts/dev

```diff
@@ -22,6 +22,37 @@ const _require = esm(module, {
   }
 })
 
+const execa = require('execa')
+
+global = new Proxy(global, {
```

Comment:
Do we really need to proxy entire global for lazy evaluating `__NUXT_VERSION_`? I mean maybe there will be easier ways for that.

Reply:
It wasnt actually about lazy evaluation but to make sure that whenever the version is used the dirtyness is correct. It would probably be fine I guess to remove the global Proxy as currently the version is only used/displayed on initial startup. But if the version would be displayed after say every dev run then the version string should be composed every time to check for correct dirtyness.

Reply:
But actually you have a good point. Please see the new commit, now we only proxy global.__NUXT